app/views/posts/index.html.erb

```diff
@@ -1,2 +1,65 @@
-<h1>Posts</h1>
-<p>Lists all posts for a certain user</p>
\ No newline at end of file
+<div class="d-flex row justify-content-center mt-5">
+  <div class= "card text-dark bg-light  p-0 m-1" style="max-width: 7rem" >
+      <div class=" border-end pr-2">
+        <%= image_tag("images.png", :alt => "rss feed", class: 'img-fluid rounded-start')   %>
+      </div>
+  </div>
+
+  <div class="card text-dark bg-light m-1 d-flex flex-row col-md-8 justify-content-between">
+    <div class="  d-flex align-items-center">
+      <h2 class="card-title text-primary"> All post written by <%= @user.name %></h2> 
+    </div>
+
+    <div class = " d-flex align-items-end justify-content-end" >
+      <p class="text-right">Number of posts: 
+        <%= @user.posts_counter ?  @user.posts_counter : 0  %>
+      </p>
+    </div>  
+  </div>
+
+  <% @user.posts.each do |post| %>
+
+  <div class = "bg-light card col-md-9 m-1  p-3 d-flex" >
+    <h3>Post# <%=  post.id %> </h3>
```

Comment:
- [ ] I am not able to see a post page to be able to see its comment see the error:
![show](https://user-images.githubusercontent.com/44745855/172484736-e050a7b8-5cc8-4123-9754-54d92178721a.png)

Reply:
Thanks @DeliceLydia, it has been resolved